content/docs/0.16.x/manage/remediation-sequence/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Remediation Sequence
+description: Implement a remediation sequence
+weight: 1
+icon: setup
+---
+
```

Comment:
Note: The evaluation sequence is not the trigger for a remediation sequence. It can be any problem that is received by Keptn. 

Can we please change the next two paragraphs from: 

```
A remediation sequence implements self-healing for a problem detected by an evaluation sequence.
It is most commonly used in production stages but can be used in any type of stage.

A remediation sequence is triggered by an event posted by an evaluation sequence.
The remediation defines the action to take to attempt to correct the problem
that the evaluation sequence identified.
```

to:

```
A remediation sequence implements self-healing for a problem detected by a monitoring solution and sent to Keptn.  The remediation defines the action to take to attempt to correct the problem.
It is most commonly used in production stages but can be used in any type of stage.
```

Reply:
Much better!  Done!